### ISSUE TITLE
Failing E2E: tabbing-order

### ISSUE THREAD

**Issue by bgambocjaviniar** [SAP/spartacus#7020]
failing constantly since the refactoring on some parts of accessibility

more info: https://jkmaster.test.c3po.b2c.ydev.hybris.com/job/spartacus-regression-tests/909/consoleText

**Comment by kacperknapik** [comment]
Duplicate of #6758
